### PR TITLE
Update OpenSSL to 1.1.1j and OpenLDAP to 2.4.57

### PR DESCRIPTION
**Pull request by KVS85**
3.6 PR: https://github.com/arangodb/arangodb/pull/13624
3.7 PR: https://github.com/arangodb/arangodb/pull/13623
devel PR: https://github.com/arangodb/arangodb/pull/13622

### CODE REVIEW

File: containers/buildAlpine3.docker/Dockerfile [hunk 1]
```diff
@@ -8,12 +8,10 @@ COPY ./tools/install-poll.sh /tools/
...
 
 COPY ./tools/install-openssl.sh /tools/
-RUN [ "/tools/install-openssl.sh", "1.1.1", "i" ]
-RUN [ "/tools/install-openssl.sh", "1.1.0", "l" ]
+RUN [ "/tools/install-openssl.sh", "1.1.1", "j" ]
 
 COPY ./tools/install-openldap.sh /tools/
-RUN [ "/tools/install-openldap.sh", "1.1.1i" ]
-RUN [ "/tools/install-openldap.sh", "1.1.0l" ]
+RUN [ "/tools/install-openldap.sh", "1.1.1j" ]
```
Comment (joerg84) [inline containers/buildAlpine3.docker/Dockerfile:14]: Just curious: why did we install 2 versions before?

**Review by joerg84 — APPROVED** [review]
@fceller Just wondering: are these risky changes and should be discussed by release owners?

File: containers/buildAlpine3.docker/Dockerfile (hunk 1 above)
Comment (KVS85) [reply containers/buildAlpine3.docker/Dockerfile:14]: We had `1.1.0` till 3.4 and just kept theoretical possibility to build even with newer Alpine.

**Review by fceller — APPROVED** [review]

**Comment by ewoutp** [comment]
General comment: I see a lot of literal config values that are the same in multiple docker files.
What about using ARGS for those in the docker files and having all the numbers in 1 central script/makefile ?

**Comment by KVS85** [comment]
> General comment: I see a lot of literal config values that are the same in multiple docker files.
> What about using ARGS for those in the docker files and having all the numbers in 1 central script/makefile ?

Yes, we can do it in future. Currently it's due to historical reasons.

**Pull request merged**